### ISSUE TITLE
plot(section) is not labelling the y axis

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1116]
Sample code:
```R
library(oce)
data(section)
plot(section, which=3, ylim=c(2000,0),xlim=c(0,400),mar=c(4,4,1,1))
```

**Comment by dankelley** [comment]
Fixed in my local version (not yet committed). It was an easy fix, partly because it was obvious to find and it was also obvious that what I was doing (purposefully blanking-out the label) made no sense *at all*.

**Comment by dankelley** [comment]
Fixed in `develop` commit 785bfab1e7b59f6a6c4b6ae8f458ecf8b1c5ece2
